### PR TITLE
Apple Silicon - M1 - added darwin-arm64 platform

### PR DESCRIPTION
**Pull request by Beekiper**
Addressing issue https://github.com/caffco/get-video-duration/issues/82

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 [![NPM bundle size (minified)][bundle-size-image]][npm-url]
 [![Downloads][downloads-image]][downloads-url]
 
-Get the duration of video files/streams with `ffprobe`.
+Get the duration of video files/streams with `ffprobe`. This package is a patch of get-video-duration with darwin-arm64 support (Apple Silicon).
```
Comment (Sumolari) [inline README.md:12]:
```suggestion
Get the duration of video files/streams with `ffprobe`.
```

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
-  "name": "get-video-duration",
-  "description": "Get the duration of a video file",
-  "version": "3.0.2",
+  "name": "get-video-duration-apple-silicon",
+  "description": "Get the duration of a video file, with darwin-arm64 support (Apple Silicon)",
+  "version": "3.0.3",
```
Comment (Sumolari) [inline package.json:4]:
```suggestion
  "name": "get-video-duration",
  "description": "Get the duration of a video file",
  "version": "3.0.2",
```

File: src/index.ts [hunk 3]
```diff
@@ -1,5 +1,7 @@
 /// <reference types="./ffprobe" />
-import { path as ffprobePath } from '@ffprobe-installer/ffprobe'
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```
Comment (Sumolari) [inline src/index.ts:3]: Why is this ignore neeed?

**Review by Sumolari — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 3 above)
Comment (Beekiper) [reply src/index.ts:3]: @Sumolari Well… I don’t know quite know TypeScript and if I don’t ts-ignore that line it doesn’t work. Any suggestions?

Comment (Sumolari) [reply src/index.ts:3]: I think you are missing changing `'@ffprobe-installer/ffprobe'` by `'node-ffprobe-installer'` in `src/ffprobe.d.ts`

Comment (Beekiper) [reply src/index.ts]: Done! Ready to merged.

**Review by Sumolari — APPROVED** [review]

**Pull request merged**